subprojects/packagefiles/libxml2/meson.build

```diff
@@ -78,9 +78,9 @@ if math_lib.found()
 endif
 
 if meson.version().version_compare('>= 0.60')
-  iconv_lib = dependency('iconv', required: false)
+  iconv_lib = dependency('iconv', required: get_option('iconv'))
 else
-  iconv_lib = dependency('', required: false)
+  iconv_lib = dependency('', required: get_option('iconv'))
```

Comment:
This doesn't work. It's a dummy never-found dependency, so building with `-Diconv=enabled` will make this error out.

That being said, this code has never actually worked before, checking for iconv support on Meson below 0.60 has always required checking cc.has_function with and without the result of cc.find_library('iconv'), and before the 0.60 support was added, this just set:

```meson
withcdata.set10('WITH_ICONV', iconv_lib.found())
```

so iconv was only configured *iff* you had an external iconv.

Reply:
Okay, I think I have the logic correct now.